package.json

```diff
@@ -50,5 +50,8 @@
         "npm-run-all": "^4.1.5",
         "prettier": "2.7.1",
         "wait-on": "^6.0.1"
+    },
+    "dependencies": {
```

Comment:
Production dependencies should **NOT** be in the project root. This can cause problems when deploying the application. This is the exact reason why express-async-handler and axios caused dependency issues for the backend.

Remove it with `npm uninstall zod`, and add it back to only frontend.